<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab1</class>
 <widget class="QWidget" name="Lab1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>600</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Phi = </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Psi = </string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <item>
       <property name="text">
        <string>Сетка X</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Сетка Y</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Сетка XY</string>
       </property>
      </item>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>1.500000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Шаг:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>212</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="minimum">
       <double>-20.000000000000000</double>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>-3.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="minimum">
       <double>-20.000000000000000</double>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="minimum">
       <double>-20.000000000000000</double>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>-3.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>252</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="minimum">
       <double>-20.000000000000000</double>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Центрирование</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Масштаб:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>10.000000000000000</double>
      </property>
      <property name="value">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Псевдо-ребра</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Lab1GLDrawer" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Lab1GLDrawer</class>
   <extends>QWidget</extends>
   <header>lab1gldrawer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
